project/Dependencies.scala

```diff
@@ -2,16 +2,23 @@ import sbt._
 
 object Dependencies {
 
+  import Versions._
```

Comment:
this is not good in practice -- it's just a matter of time until the names will clash

Reply:
Right, I'll remove it.